xmrstak/backend/amd/amd_gpu/gpu.hpp

```diff
@@ -4,6 +4,9 @@
 
 #if defined(__APPLE__)
 #include <OpenCL/cl.h>
+#include <string>
+#include <sstream>
+#include <iostream>
```

Comment:
this is not needed; there already is an include of "string" at line 15 (it was added at commit 406cf0f)

Reply:
For the record: it was fixed at #194 already merged to dev 16 days ago.

Reply:
Thanks! @auricgoldfinger - can you confirm if the dev branch is compiling ok for you without the pr?

Reply:
Yes, true. `sstream` and `iostream` aren't needed to build and `string` was indeed included.
I admit I changed it on master and discovered only later that pull requests must be done on dev, so I rebased it.

Only question that remains: is this included needed only `if defined(__APPLE__)` or always?